lib/models/autodecoders/base_nerf.py

```diff
@@ -414,6 +439,7 @@ def inverse_code(self, decoder, cond_imgs, cond_rays_o, cond_rays_d, dt_gamma=0,
         with module_requires_grad(decoder, False):
             n_inverse_steps = cfg.get('n_inverse_steps', 1000)
```

Comment:
This is probably enough, but let's double check if` code` doesn't require copying and detaching.